packages/compose/src/hooks/use-resize-observer/index.native.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { View, StyleSheet } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
+
+/**
+ * Hook which allows to listen the resize event of any target element when it changes sizes.
+ *
+ * @return {Object} Measurements object with properties `width` and `height`.
```

Comment:
This documentation block should be identical to the web one

Reply:
fixed!